.github/workflows/ci.yml

```diff
@@ -92,6 +92,8 @@ jobs:
       - run: yarn install --ignore-engines
         # You can run `yarn fix-lint` to fix all Prettier complaints.
       - run: yarn lint
+        # Make sure yarn.lock did not change with `yarn install`
+      - run: git diff --stat --exit-code -- yarn.lock
```

Comment:
Perfect; thanks! Extra points for the extra safe `-- yarn.lock` form,
and thanks for verifying that it fails at head.